elasticdl/python/tests/worker_ps_interaction_test.py

```diff
@@ -31,38 +32,75 @@ def random_batch(batch_size):
     return images, labels
 
 
+def get_mnist_dataset(batch_size):
+    (
+        (x_train, y_train),
+        (x_test, y_test),
+    ) = tf.keras.datasets.mnist.load_data()
+    x_train = tf.convert_to_tensor(x_train, dtype=tf.float32) / 255.0
+    y_train = tf.convert_to_tensor(y_train, dtype=tf.int32)
+
+    x_test = tf.convert_to_tensor(x_test, dtype=tf.float32) / 255.0
+    y_test = tf.convert_to_tensor(y_test, dtype=tf.int32)
+
+    db = tf.data.Dataset.from_tensor_slices((x_train, y_train))
+    db = db.batch(batch_size).repeat(2)
+    test_db = tf.data.Dataset.from_tensor_slices((x_test, y_test))
+    test_db = test_db.batch(batch_size)
+
+    return db, test_db
+
+
+def get_frappe_dataset(batch_size):
+    home = str(Path.home())
+
+    class TmpArgs(object):
+        def __init__(self, data):
+            self.data = data
+
+    args = TmpArgs(data=home + "/.keras/datasets/")
```

Comment:
Use `path.join` instead of string concatenation would be better

Reply:
Done